src/spaces/graph.jl

```diff
@@ -91,7 +90,13 @@ nodes(model::ABM{<:AbstractAgent,<:GraphSpace}) = 1:nv(model)
 #######################################################################################
 function space_neighbors(pos::Int, model::ABM{A,<:GraphSpace}, args...; kwargs...) where {A}
     nn = node_neighbors(pos, model, args...; kwargs...)
-    Iterators.flatten(model.space.s[i] for i in Iterators.flatten((pos,nn)))
+    # This call is faster than reduce(vcat, ..), or Iterators.flatten
+    vcat(model.space.s[pos], model.space.s[nn]...)
+end
+
+function space_neighbors(agent::A, model::ABM{A,<:GraphSpace}, args...; kwargs...) where {A<:AbstractAgent}
```

Comment:
Why did you have to explicitly add this method? Isn't it defined already in the space API?

Reply:
No, that uses `Iterators.filter`, which is good for when you want to filter on an existing iterator (used in `GridSpace`), but if you want to drop something out of a `Vector`, `filter!` is faster.

Reply:
Of course, awesome.